packages/sui-bundler/shared/module-rules-babel.js

```diff
@@ -1,6 +1,8 @@
+const {sep} = require('path')
+
 module.exports = {
   test: /\.jsx?$/,
-  exclude: /node_modules(?!\/@s-ui\/studio\/src)/,
+  exclude: new RegExp(`node_modules(?!${sep}@s-ui${sep}studio${sep}src)`),
```

Comment:
This change had been un done but ends up being necessary. Probably sth that has to do with linking.